imagetest/test_suites/metadata/metadata_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build cit
 // +build cit
```

Comment:
You removed `+build cit` in another file but left it here; also in another file, you only have  `+build cit` . What's the difference between `//go:build cit` and `// +build cit`>

Reply:
I don't quite understand the diff, seems introduced by go check. I remove this as no reason to make change.

Reply:
lgtm